osc-service-api/src/main/java/org/osc/core/broker/service/dto/ApplianceManagerConnectorDto.java

```diff
@@ -135,7 +135,7 @@ public void setLastJobId(Long lastJobId) {
         this.lastJobId = lastJobId;
     }
 
-    public Boolean isPolicyMappingSupported() {
+    public Boolean getIsPolicyMappingSupported() {
```

Comment:
line comment to indicate why the non-standard naming, just in case someone comes in and "fixes" this and causes regressions